browser/about_flags.cc

```diff
@@ -266,24 +262,22 @@ const char* const kBraveSyncImplLink[1] = {"https://github.com/brave/go-sync"};
           FEATURE_VALUE_TYPE(brave_news::features::kBraveNewsFeedUpdate),      \
       })
 
-#define PLAYLIST_FEATURE_ENTRIES                                       \
-  IF_BUILDFLAG(                                                        \
-      ENABLE_PLAYLIST,                                                 \
-      EXPAND_FEATURE_ENTRIES(                                          \
-          {                                                            \
-              "playlist",                                              \
-              "Playlist",                                              \
-              "Enables Playlist",                                      \
-              kOsMac | kOsWin | kOsLinux | kOsAndroid,                 \
-              FEATURE_VALUE_TYPE(playlist::features::kPlaylist),       \
-          },                                                           \
-          {                                                            \
-              "playlist-fake-ua",                                      \
-              "PlaylistFakeUA",                                        \
-              "Use fake UA for playlist",                              \
-              kOsMac | kOsWin | kOsLinux | kOsAndroid,                 \
-              FEATURE_VALUE_TYPE(playlist::features::kPlaylistFakeUA), \
-          }))
+#define PLAYLIST_FEATURE_ENTRIES                                   \
```

Comment:
does these still need a feature specific define if there's no associated buildflag? Can't we just list it out normally?

Reply:
I see a number of feature lists not associated with buildflags in the file (e.g. `BRAVE_TABS_FEATURE_ENTRIES`, `BRAVE_NEWS_FEATURE_ENTRIES`, `BRAVE_NATIVE_WALLET_FEATURE_ENTRIES`, etc.)